recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 package:
   name: conda
-  version: {{ GIT_DESCRIBE_TAG }}+{{ GIT_BUILD_STR }}
```

Comment:
Interesting that `conda-build` already [had a dot instead of a plus sign](https://github.com/conda/conda-build/pull/5483/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aL3)!